go.mod

```
@@ -2,7 +2,7 @@ module k8s.io/ingress-nginx
 
 go 1.22.0
```

maybe you can bump here

```suggestion
go 1.22.4
```

---

hmm, ill fix it in the golang update